ibis/backends/base_sqlalchemy/mysql/api.py

```diff
@@ -16,7 +16,7 @@ def compile(expr, params=None):
 
     Returns
     -------
-    sqlalchemy_expression : sqlalchemy.sql.expression.ClauseElement
+    sqlalchemy_expression : sqlalchemy.base_sqlalchemy.expression.ClauseElement
```

Comment:
This is the sqalchemy module (a third-party package), which obviously didn't change.